app/uk/gov/hmrc/apigatekeeperapisfrontend/models/EnvironmentDefinitions.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.apigatekeeperapisfrontend.models
+
+import uk.gov.hmrc.apiplatform.modules.apis.domain.models.ApiDefinition
+
+case class EnvironmentDefinitions(sandbox: List[ApiDefinition], production: List[ApiDefinition]) {
```

Comment:
This class name is confusing. It sounds like there's something environmental about it but I don't think there is, as it's merging the lists from sandbox and production.  Maybe `AllDefinitions`?

Reply:
Well its not merging them by default it's mostly to keep the two lists with env details on them. It's to replace an Environment -> List[ApiDef] map as we only have 2 keys (prod and sandbox)